CRM/Mailing/Selector/Browse.php

```diff
@@ -409,6 +403,10 @@ public function &getRows($action, $offset, $rowCount, $sort, $output = NULL) {
           }
         }
 
+        if ($allAccess || $showCreateLinks) {
+          $actionMask |= CRM_Core_Action::UPDATE;
+          }
```

Comment:
this indentation is going to cause jenkins to scream blue murder - tests are running now so it looks like

![image](https://user-images.githubusercontent.com/336308/114806194-6eb8ff00-9df8-11eb-8283-2bee6d105434.png)


But I'm betting the brown circle turns into a red x - you can click through on the x to see what jenkins has to say

Reply:
fixed